src/gatsby/models/docs/onCreateMarkdownContentNode.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path')
+
+function createMarkdownDocsNode(api, { parentNode }) {
+  const splitDir = parentNode.relativeDirectory.split('/')
+  if (splitDir[0] !== 'docs') return
+
+  const { node, actions, createNodeId, createContentDigest } = api
+  const { createNode, createParentChildLink } = actions
+  const { name, relativePath } = parentNode
+  splitDir[0] = 'doc'
+
+  // Make a special exemption for the root doc.
+  const slug =
```

Comment:
I'm not sure this is correct - slug comes from the sidebard.json for docs, not from the file system

Reply:
it is correct, slug is the same as on the file system